src/main/java/com/khutircraftubackend/auth/MarketingCampaignEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.khutircraftubackend.auth;
```

Comment:
probably not the best place for this class

Reply:
Created a package marketing and moved the Entity and Repository for MarketingCampaign into it.